javers-core/src/main/java/org/javers/core/metamodel/type/ManagedClassFactory.java

```diff
@@ -4,10 +4,7 @@
 import org.javers.common.exception.JaversException;
 import org.javers.common.exception.JaversExceptionCode;
 import org.javers.core.metamodel.annotation.ClassAnnotationsScanner;
-import org.javers.core.metamodel.clazz.ClientsClassDefinition;
-import org.javers.core.metamodel.clazz.EntityDefinition;
-import org.javers.core.metamodel.clazz.ValueDefinition;
-import org.javers.core.metamodel.clazz.ValueObjectDefinition;
+import org.javers.core.metamodel.clazz.*;
```

Comment:
mi to jakoś mocno nie przeszkadza, ale wydaje mi sie, że zazwyczaj staraliśmy się unikać star imports